src/main/webapp/app/overview/exercise-details/exercise-details-student-actions.component.ts

```diff
@@ -231,11 +231,9 @@ export class ExerciseDetailsStudentActionsComponent {
     public shouldDisplayIDEButtons(): boolean {
         return !!this.exercise.studentParticipations?.some((participation) => {
             const status = participationStatus(this.exercise, participation.testRun);
```

Comment:
This is an accidental refactoring as I was looking in the wrong place in the code.
It should do exactly the same as before